tests/analysis_tests/tests/src/expected/Fragment.res.txt

```diff
@@ -3,14 +3,8 @@ Hover src/Fragment.res 6:19
 
 Hover src/Fragment.res 9:56
 Nothing at that position. Now trying to use completion.
-posCursor:[9:56] posNoWhite:[9:55] Found expr:[9:10->9:67]
-posCursor:[9:56] posNoWhite:[9:55] Found expr:[9:13->9:67]
+posCursor:[9:56] posNoWhite:[9:55] Found expr:[9:9->11:0]
 posCursor:[9:56] posNoWhite:[9:55] Found expr:[9:13->9:66]
 JSX <SectionHeader:[9:13->9:26] > _children:9:26
-posCursor:[9:56] posNoWhite:[9:55] Found expr:__ghost__[9:10->9:67]
```

Comment:
This ghost expression is a part of the `AstHelper.make_list_expression` result.
It is no longer present in the new AST, but it also didn't serve any purpose.
I believe it is okay that this test is slightly different.
In the end the result didn't change.